### DISCUSSION TITLE
What about request journal?

### DISCUSSION THREAD

**Discussion by ushankax** [bbyars/mountebank#740] [Ideas]
Hi! We recently decided to switch from Hoverfly to Mountebank because of gRPC plugin. So the advantage of Hoverfly was in possibility to get journal with history of requests. In Mountebank we have `/logs` only and its not so comfortable to debug.
 
Are any ways to get request journal in mb? Thanks!

**Comment by bbyars** [comment]
Hi Sergey,
Yes. If you set "recordRequests" to true at the imposter header level (e.g.
as a sibling to "protocol" and "port"), then mountebank records the
requests for that imposter. You can retrieve them by getting the imposter
resource (https://www.mbtest.org/docs/api/mocks). You can also get a bit
more data by running mb with the --debug flag
<https://www.mbtest.org/docs/commandLine#start>.
-Brandon

**Comment by ushankax** [comment]
Thank you! This is it!
